app/cdap/components/WrangleHome/Components/OngoingDataExplorations/Constants.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export const CONNECTOR_TYPE = 'connectorType';
+export const ICON = 'icon';
+export const CONNECTION_NAME = 'connectionName';
+export const WORKPSACE_NAME = 'workspaceName';
+export const ICON_WITH_TEXT = 'iconWithText';
+export const TEXT = 'text';
+export const RECIPE_STEPS = 'recipeSteps';
+export const DATA_QUALITY = 'dataQuality';
+export const NULL_VALUES = 'Null values';
```

Comment:
move this null values to text-en as it is used to display text on the DOM

Reply:
done